contrib/python-hypothesis/template.py

```diff
@@ -0,0 +1,39 @@
+pkgname = "python-hypothesis"
+pkgver = "6.92.2"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+checkdepends = [
+    "python-attrs",
+    "python-pexpect",
+    "python-pytest",
+    "python-pytest-xdist",
+    "python-sortedcontainers",
+]
+depends = ["python-attrs", "python-sortedcontainers"]
+pkgdesc = "Easy to use library for property-based testing"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "MPL-2.0"
+url = "https://hypothesis.works/index.html"
+source = f"https://github.com/HypothesisWorks/hypothesis/archive/refs/tags/hypothesis-python-{pkgver}.zip"
```

Comment:
.tar.gz

Reply:
I think they were different errors, so harder to bunch into fewer ignores